go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/aws/aws-sdk-go v1.44.189
-	github.com/aws/aws-sdk-go-v2 v1.19.0
+	github.com/aws/aws-sdk-go-v2 v1.21.0
```

Comment:
@madhushreeray30, we are updating the AWS SDK version `1.19.0` to `1.21.0`. Are there any significant changes or breaking changes between these two versions that we should be aware of?